protos/monocle/http.proto

```diff
@@ -91,25 +91,25 @@ service UserGroup {
 
 service TaskData {
   // Commit the last updated timestamp
-  rpc Commit(monocle_task_data.TaskDataCommitRequest) returns (monocle_task_data.TaskDataCommitResponse) {
+  rpc TaskDataCommit(monocle_task_data.TaskDataCommitRequest) returns (monocle_task_data.TaskDataCommitResponse) {
```

Comment:
Where are we now prefixing rpc and message name?

Reply:
I've got hard time to avoid conflict with Crawler's CommitRequest/CommitResponse. The solution I found was this rpc call renaming. I'm open to other solutions if any.

Reply:
I see, then I guess we could improve the codegen to use qualified import (in a follow-up).